src/Logic/CalcLengthOfCart.js

```diff
@@ -0,0 +1,7 @@
+export const calculateLengthOfCart = (cart) => {
+  let length = 0;
+  cart.map((item) => {
+    length += item.quantity;
+  });
+  return length;
```

Comment:
Just a suggestion: I have seen other ecomm site, and they show cart size based on variety of products and not the quantities added for each product.